setup.py

```diff
@@ -1,10 +1,46 @@
 #!/usr/bin/env python
+import json
 import logging
 import os
 from importlib import util
 from os import path
+from pathlib import Path
 
 from setuptools import setup, find_packages
+from setuptools.command.build_py import build_py
+
+
+class PreBuildCommand(build_py):
+    """Pre-build command"""
+
+    def transform_graph_yaml_to_json(self) -> None:
+        """Transforms YAML graph checks to JSON and copies them to build/lib"""
+
+        import yaml  # can't be top-level, because it needs to be first installed via 'setup_requires'
+
+        graph_check_paths = (
+            "checkov/ansible/checks/graph_checks",
+            "checkov/arm/checks/graph_checks",
+            "checkov/bicep/checks/graph_checks",
+            "checkov/cloudformation/checks/graph_checks",
+            "checkov/dockerfile/checks/graph_checks",
+            "checkov/github_actions/checks/graph_checks",
+            "checkov/kubernetes/checks/graph_checks",
+            "checkov/terraform/checks/graph_checks",
+        )
```

Comment:
How about iterating all `checkov/*/checks/graph_checks` paths to find the ones that exist? It'll prevent missing adding paths to that list